lib/data/landing_page_content_items/task.yaml

```diff
@@ -48,7 +48,6 @@ blocks:
       x_axis_label: "X Axis"
       y_axis_label: "Y Axis"
       csv_file: "data_one.csv"
-      data_source_link_text: "Data source"
```

Comment:
If this label is no longer needed, it should be removed from the other demos as well as the tests and the fixtures.

Perhaps that will be tidied up in a later PR?

Reply:
Good shout, have raised: https://github.com/alphagov/frontend/pull/4313